src/org/opendatakit/briefcase/ui/reused/UI.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.ui.reused;
+
+import static java.awt.Color.LIGHT_GRAY;
+import static javax.swing.JOptionPane.getFrameForComponent;
+import static org.opendatakit.briefcase.ui.ScrollingStatusListDialog.showDialog;
+
+import java.awt.Font;
+import java.awt.Insets;
+import javax.swing.JButton;
+import javax.swing.JTable;
+import org.opendatakit.briefcase.model.FormStatus;
+
+public class UI {
```

Comment:
I know it will look funny, but since “UI” is an initialism, in an identifier it should be “Ui”.